### PR TITLE
bootstrapping button-group component

### PR DESCRIPTION
**Pull request by sqevani**
task: button-group changes the styles of how the buttons look, and groups them together

screenshot 2018-08-29 14 46 10

### CODE REVIEW

**Comment by sadiqevani** [comment]
@davidyuk I've implemented the button group, up for review.

File: src/components/ae-button-group/ae-button-group.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+```vue
+  <ae-button-group>
+    <ae-button>Button</ae-button>
+    <ae-button>Group</ae-button>
+  </ae-button-group>
+``` 
+
+### prop: face
```
Comment (davidyuk) [inline src/components/ae-button-group/ae-button-group.md]: AeButtonGroup doesn't have a `face` property.

File: src/components/ae-button-group/ae-button-group.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    <ae-button face="round" fill="neutral">Button</ae-button>
+    <ae-button face="round" fill="neutral">Group</ae-button>
+  </ae-button-group>
+``` 
+
+```vue
+  <ae-button-group>
+    <ae-button face="round" fill="alternative">Button</ae-button>
+    <ae-button face="round" fill="alternative">Group</ae-button>
+  </ae-button-group>
```
Comment (davidyuk) [inline src/components/ae-button-group/ae-button-group.md]: Documentation is not a tests, so I think only one example with ` <ae-button face="round">` is enough. All possible colors of AeButton are already included in the documentation of AeButton component.

File: src/components/ae-button-group/ae-button-group.vue [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+  /deep/ .ae-button {
+    border-radius: 0;
+    border: 0;
+    box-shadow: none;
+    flex-grow: 1;
+
+    &.round {
+      border-left-width: 1px;
+      border-right-width: 1px;
+      border-style: solid;
```
Comment (davidyuk) [inline src/components/ae-button-group/ae-button-group.vue]: https://app.zeplin.io/project/59e4cf99fc0bb2f99a89551b/screen/5b97e915ac40214afbbdc55b
Actually, I can't see any border between buttons on this screen.

Comment (sadiqevani) [reply src/components/ae-button-group/ae-button-group.vue]: @davidyuk fixed.

**Comment by sadiqevani** [comment]
@davidyuk 

I've reverted back some of your style changes, currently the component supports only round button grouping, but that does not mean it wont support other buttons.

In terms of styles, please leave that to me, if you think some styles should be different just let me know.

**Pull request closed without merging**